src/Core/Domain/MerchandiseReturn/Command/BulkDeleteProductFromMerchandiseReturnCommand.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\Exception\MerchandiseReturnConstraintException;
+use PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\Exception\MerchandiseReturnException;
+use PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\ValueObject\MerchandiseReturnDetail;
+use PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\ValueObject\MerchandiseReturnId;
+
+/**
+ * Deletes products from given order.
```

Comment:
```suggestion
 * Deletes products from given order return.
```

By the way, what does it mean "deleting products from an order return" ? 🤔 
I know we can do it in the UX, but when does it happen in a e-commerce lifecycle ?

Customer orders product A
Customer pays product A
Customer is not happy with product A
Customer returns product A
...
So when we "delete a product from an order return" do we ship back product A to customer who changed his opinion ?" 😆

Reply:
Maybe customer ordered Product A, Product B, and Product C.

Product B is not eligable for return, so we send it back? (not sure how it works), Product C is not actually in package with returned products. So we delete those 2 and only return money from Product A?

Reply:
indeed this is relevant scenario